### ISSUE TITLE
Delayed processing of backups in status "New"

### ISSUE THREAD

**Issue by pdstefan** [vmware-tanzu/velero#5847]
**What steps did you take and what happened:**
After upgrading to velero 1.10, nightly backups which were processed in ~2 hours now take 8h+. 
While debugging we observed backups in phase NEW get executed in batches of 10 every 5 minutes, which causes unnecessary delays. 


**What did you expect to happen:**
As long as "New" backups exist, trigger backup continuously (or at least with greater frequency) until no "New" backups exist. 

**The following information will help us better understand what's going on**:

Our debug steps are as follows: 

1. 12:01:53 previous batch finished processing "New" backups.
2. 174 "New" backups in queue: 

```
date ; velero backup get | grep New | grep -v "realtime" |  cut -d" " -f 1 | wc -l 
Wed Feb  8 12:05:24 CET 2023                                                                                                                                                                                                                                           
174                

```
```
time="2023-02-08T12:01:53+01:00" level=info msg="Skipping snapshot of persistent volume because volume is being backed up with pod volume backup." backup=velero/backup-ora-0e07712-20230207210059 logSource="pkg/backup/item_backupper.go:430" 
name=pvc-1670b32b-3376-4835-a78a-1452703f48d5 namespace= persistentVolume=pvc-1670b32b-3376-4835-a78a-1452703f48d5 resource=persistentvolumes

time="2023-02-08T12:01:59+01:00" level=info msg="Initializing backup repository" backupRepo=velero/ora-0d01676-default-restic-rf5gg logSource="pkg/controller/backup_repository_controller.go:113"
time="2023-02-08T12:01:59+01:00" level=info msg="Set user defined maintenance frequency" frequency=2h0m0s logSource="pkg/controller/backup_repository_controller.go:160"
time="2023-02-08T12:03:13+01:00" level=info msg="Validating BackupStorageLocation" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:151"
time="2023-02-08T12:03:13+01:00" level=info msg="BackupStorageLocations is valid, marking as available" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:136"
```

From observed behavior next pool for "New" backups should be at : 12:06:53(+/-10sec) 

``` 
time="2023-02-08T12:03:13+01:00" level=info msg="Validating BackupStorageLocation" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:151"
time="2023-02-08T12:03:13+01:00" level=info msg="BackupStorageLocations is valid, marking as available" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:136"
time="2023-02-08T12:06:13+01:00" level=info msg="Validating BackupStorageLocation" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:151"
time="2023-02-08T12:06:13+01:00" level=info msg="BackupStorageLocations is valid, marking as available" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:136"
===> 
time="2023-02-08T12:06:53+01:00" level=info msg="1 errors encountered backup up item" backup=velero/backup-ora-0e07712-20230207210059 logSource="pkg/backup/backup.go:421" name=ora-0e07712-57699fd97c-8h2dq
time="2023-02-08T12:06:53+01:00" level=error msg="Error backing up item" backup=velero/backup-ora-0e07712-20230207210059 error="failed to wait BackupRepository: timed out waiting for the condition" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/repository/ensurer.go:129" error.function="github.com/vmware-tanzu/velero/pkg/repository.(*RepositoryEnsurer).createBackupRepositoryAndWait" logSource="pkg/backup/backup.go:425" name=ora-0e07712-57699fd97c-8h2dq
time="2023-02-08T12:06:53+01:00" level=info msg="Backed up 3 items out of an estimated total of 31 (estimate will change throughout the backup)" backup=velero/backup-ora-0e07712-20230207210059 logSource="pkg/backup/backup.go:388" name=ora-0e07712-57699fd97c-8h2dq namespace=ora-0e07712 progress= resource=pods
``` 

As observed previously, 5 minutes after the actions from the previous batch completed, a new batch of ~10 backups is processed. 

``` 
date ; velero backup get | grep New | grep -v "realtime" |  cut -d" " -f 1 | wc -l 
Wed Feb  8 12:10:58 CET 2023                                                                                                                                                                                                                                           
167 
``` 
And once more: 12:07:30 marks the end on previous "batch", adding 5 minutes we see another batch  being processed. We can see the backupstorage validation messages (every 3 minutes) in between batches. 

```
time="2023-02-08T12:07:30+01:00" level=info msg="Skipping snapshot of persistent volume because volume is being backed up with pod volume backup." backup=velero/backup-ora-0m05576-20230207210100 logSource="pkg/backup/item_backupper.go:430" name=pvc-f0b19c8f-d395-4877-a20b-4bf8d2d2a481 namespace= persistentVolume=pvc-f0b19c8f-d395-4877-a20b-4bf8d2d2a481 resource=persistentvolumes
time="2023-02-08T12:09:13+01:00" level=info msg="Validating BackupStorageLocation" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:151"
time="2023-02-08T12:09:13+01:00" level=info msg="BackupStorageLocations is valid, marking as available" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:136"
===> 
time="2023-02-08T12:12:05+01:00" level=info msg="Initializing backup repository" backupRepo=velero/ora-0m05576-default-restic-6rcl9 logSource="pkg/controller/backup_repository_controller.go:113"
time="2023-02-08T12:12:05+01:00" level=info msg="Set user defined maintenance frequency" frequency=2h0m0s logSource="pkg/controller/backup_repository_controller.go:160"
time="2023-02-08T12:12:13+01:00" level=info msg="Backed up 3 items out of an estimated total of 30 (estimate will change throughout the backup)" backup=velero/backup-ora-0m05576-20230207210100 logSource="pkg/backup/backup.go:388" name=ora-0m05576-55768f5c5c-j4clt namespace=ora-0m05576 progress= resource=pods
``` 

``` 
time="2023-02-08T12:12:18+01:00" level=info msg="Executing takePVSnapshot" backup=velero/backup-ora-0g41320-20230207210053 logSource="pkg/backup/item_backupper.go:412" name=pvc-f47479b0-b7d9-41cf-9f60-545e297172ba namespace= resource=persistentvolumes
time="2023-02-08T12:12:18+01:00" level=info msg="Skipping snapshot of persistent volume because volume is being backed up with pod volume backup." backup=velero/backup-ora-0g41320-20230207210053 logSource="pkg/backup/item_backupper.go:430" name=pvc-f47479b0-b7d9-41cf-9f60-545e297172ba namespace= persistentVolume=pvc-f47479b0-b7d9-41cf-9f60-545e297172ba resource=persistentvolumes
time="2023-02-08T12:15:13+01:00" level=info msg="Validating BackupStorageLocation" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:151"
time="2023-02-08T12:15:13+01:00" level=info msg="BackupStorageLocations is valid, marking as available" backup-storage-location=velero/default controller=backup-storage-location logSource="pkg/controller/backup_storage_location_controller.go:136"
====>
time="2023-02-08T12:17:06+01:00" level=info msg="Initializing backup repository" backupRepo=velero/ora-0g41320-default-restic-kb96r logSource="pkg/controller/backup_repository_controller.go:113"
time="2023-02-08T12:17:06+01:00" level=info msg="Set user defined maintenance frequency" frequency=2h0m0s logSource="pkg/controller/backup_repository_controller.go:160"
time="2023-02-08T12:17:09+01:00" level=info msg="Backed up 3 items out of an estimated total of 31 (estimate will change throughout the backup)" backup=velero/backup-ora-0g41320-20230207210053 logSource="pkg/backup/backup.go:388" name=ora-0g41320-655f586997-9x4rx namespace=ora-0g41320 progress= resource=pods

``` 
After batch number 3 we now have : 127 "New" backups in the queue, which matches the ~10 items / batch observed behavior. 
```
date ; velero backup get | grep New | grep -v "realtime" |  cut -d" " -f 1 | wc -l 
Wed Feb  8 12:20:06 CET 2023                                                                                                                                                                                                                                           
127                         
``` 
[Full log file]( https://gist.github.com/pdstefan/5cc82d7e8c3d90b0c76164f358e3c276). 


**Anything else you would like to add:**

We upgraded from velero 1.8.0 to 1.10.0 a few days ago, this is when the problems started. (later I pulled 1.10.1 hoping this would include fixes) 

We backup 1000+ namespaces with restic (a median of 500MB per namespace). We do this using schedules that are triggered to start at 21:00. Previous behavior was pretty stable and the 1000 schedules were done in ~2 hours. We also have tools that also make use of velero to generate realtime backsups and we update these 1000+ namespaces starting at 12:00 AM until 6AM (nightly maintenance window). 

When the scheduled backups were done until 12 AM, we had ~6 hours for maintenance but at this point there is no telling when the schedules will finish. For 1000 schedules, doing 10 every 5 minutes it would take ~8hours ( :( ) 

Helm values file used for upgrade: 

```
# @note - velero 1.7.x works with aws plugin 1.3.x
# velero 1.8.x works with aws plugin 1.4.x
initContainers:
  - name: velero-plugin-for-aws
    image: velero/velero-plugin-for-aws:v1.5.2
    imagePullPolicy: IfNotPresent
    volumeMounts:
      - mountPath: /target
        name: plugins

image:
  repository: velero/velero
  tag: v1.10.1

configuration:
  provider: aws
  backupStorageLocation:
    bucket: bucket
    config:
     region: eu-de-02
     s3ForcePathStyle: "true"
     s3Url: http://minio.minio.svc.cluster.local:9000

  # @note - enabling without restic being deployed blocks backups from completing
  defaultVolumesToRestic: true
  backupSyncPeriod: 1m
  clientBurst: 100
  clientQPS: 100
  #clientPageSize: "0"
  storeValidationFrequency: 3m
  defaultRepoMaintainFrequency: 2h

credentials:
  useSecret: true
  secretContents:
    cloud: |
      [default]
      aws_access_key_id = xxxxxxxxxxxxxxxxxx
      aws_secret_access_key = xxxxxxxxxxxxxxxxxxxx

  # If there is an already existing secret
  # extraSecretRef: cloud-credentials

#deployRestic: true
snapshotsEnabled: false
deployNodeAgent: true

resources:
  requests:
    cpu: 500m
    memory: 128Mi
  limits:
    cpu: 2
    memory: 900Mi

#restic:
nodeAgent:
  resources:
    requests:
      cpu: 200m
      memory: 200Mi
    limits:
      cpu: 2
      memory: 1000Mi
``` 

**Environment:**

- Velero version (use `velero version`): 
``` 
Client:
        Version: v1.10.1
        Git commit: 948b3790d5577dc7e77410a8b80ef7d8a6b00303
Server:
        Version: v1.10.1
``` 
- Velero features (use `velero client config get features`): 
```
features: <NOT SET>
``` 
- Kubernetes version (use `kubectl version`):
``` 
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.0", GitCommit:"c2b5237ccd9c0f1d600d3072634ca66cefdf272f", GitTreeState:"clean", BuildDate:"2021-08-04T18:03:20Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"23+", GitVersion:"v1.23.4-r0-CCE22.3.1.B012", GitCommit:"dbafdba319fbe5a0b328f5c1705793142ff15948", GitTreeState:"clean", BuildDate:"2022-03-26T10:13:16Z", GoVersion:"go1.17.7", Compiler:"gc", Platform:"linux/amd64"}
``` 
- Kubernetes installer & version:
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`): EulerOS SP9


**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by reasonerjt** [comment]
@pdstefan 
Thanks for writing this up.

This might be caused by the refactor made in the controllers (schedule, podvolume, backup repository...)
To help us further investigate, could you please clarify do you have these schedules set to daily and each one of them maps to one namespace?
Are they supposed to trigger backup at the same time?

**Comment by Lyndon-Li** [comment]
By checking the log, I found the problem has something to do with the backup repo. Before running a pod volume backup, Velero first tries to connect to the backup repo. It creates the backuprepository CR and the backuprepository controller connects to the backup repo and then set the CR to Ready in its reconciler. After creating the backuprepository CR, Velero waits for the CR comes to Ready until a timeout, 5 minutes.

From the log:
Sometimes, the backuprepository CR was not processed at all by the controller, until the timeout:
```
time="2023-02-08T11:56:28+01:00" level=info msg="Skipping snapshot of persistent volume because volume is being backed up with pod volume backup." backup=velero/backup-ora-0d01676-20230207210048 logSource="pkg/backup/item_backupper.go:430" name=pvc-0aa18e1d-a7e6-4853-be0b-dda9d40a5b08 namespace= persistentVolume=pvc-0aa18e1d-a7e6-4853-be0b-dda9d40a5b08 resource=persistentvolumes
....
time="2023-02-08T12:01:28+01:00" level=info msg="1 errors encountered backup up item" backup=velero/backup-ora-0d01676-20230207210048 logSource="pkg/backup/backup.go:421" name=ora-0d01676-65cc5f85cf-ppvz9
time="2023-02-08T12:01:28+01:00" level=error msg="Error backing up item" backup=velero/backup-ora-0d01676-20230207210048 error="failed to wait BackupRepository: timed out waiting for the condition" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/repository/ensurer.go:129" error.function="github.com/vmware-tanzu/velero/pkg/repository.(*RepositoryEnsurer).createBackupRepositoryAndWait" 

```
Sometimes, the backuprepository CR was processed immediately and the controller tried to initialize the repo, which is the connection process, but it didn't finish it until the timeout:
```
time="2023-02-08T12:01:53+01:00" level=info msg="Skipping snapshot of persistent volume because volume is being backed up with pod volume backup." backup=velero/backup-ora-0e07712-20230207210059 logSource="pkg/backup/item_backupper.go:430" name=pvc-1670b32b-3376-4835-a78a-1452703f48d5 namespace= persistentVolume=pvc-1670b32b-3376-4835-a78a-1452703f48d5 resource=persistentvolumes
time="2023-02-08T12:01:59+01:00" level=info msg="Initializing backup repository" backupRepo=velero/ora-0d01676-default-restic-rf5gg logSource="pkg/controller/backup_repository_controller.go:113"
time="2023-02-08T12:01:59+01:00" level=info msg="Set user defined maintenance frequency" frequency=2h0m0s logSource="pkg/controller/backup_repository_controller.go:160"
...
time="2023-02-08T12:06:53+01:00" level=info msg="1 errors encountered backup up item" backup=velero/backup-ora-0e07712-20230207210059 logSource="pkg/backup/backup.go:421" name=ora-0e07712-57699fd97c-8h2dq
time="2023-02-08T12:06:53+01:00" level=error msg="Error backing up item" backup=velero/backup-ora-0e07712-20230207210059 error="failed to wait BackupRepository: timed out waiting for the condition" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/repository/ensurer.go:129" error.function="github.com/vmware-tanzu/velero/pkg/repository.(*RepositoryEnsurer).createBackupRepositoryAndWait" logSource="pkg/backup/backup.go:425" name=ora-0e07712-57699fd97c-8h2dq
```

Sometimes, the backuprepository CR was not processed until near timeout, then the backup succeeded in the edge of failure due to the same timeout:
```
time="2023-02-08T12:07:30+01:00" level=info msg="Skipping snapshot of persistent volume because volume is being backed up with pod volume backup." backup=velero/backup-ora-0m05576-20230207210100 logSource="pkg/backup/item_backupper.go:430" name=pvc-f0b19c8f-d395-4877-a20b-4bf8d2d2a481 namespace= persistentVolume=pvc-f0b19c8f-d395-4877-a20b-4bf8d2d2a481 resource=persistentvolumes
...
time="2023-02-08T12:12:05+01:00" level=info msg="Initializing backup repository" backupRepo=velero/ora-0m05576-default-restic-6rcl9 logSource="pkg/controller/backup_repository_controller.go:113"
time="2023-02-08T12:12:05+01:00" level=info msg="Set user defined maintenance frequency" frequency=2h0m0s logSource="pkg/controller/backup_repository_controller.go:160"
time="2023-02-08T12:12:13+01:00" level=info msg="Backed up 3 items out of an estimated total of 30 (estimate will change throughout the backup)" backup=velero/backup-ora-0m05576-20230207210100 logSource="pkg/backup/backup.go:388" name=ora-0m05576-55768f5c5c-j4clt namespace=ora-0m05576 progress= resource=pods
```

**Comment by Lyndon-Li** [comment]
@pdstefan Looks like the problem is related to your env, and we need to know what happened to the backup repo connection. Please do below steps and share more info to us:
1. If you can reproduce the problem, please set Velero's log level to `debug` and recollect the log. The connection logs are all in debug level. You can follow [this guide](https://velero.io/docs/v1.10/troubleshooting/#:~:text=more%20usage%20details.-,Getting%20velero%20debug%20logs,-You%20can%20increase) to enable the debug log.
2. Please share the info of backuprepository CRs in your env, specifically, collect the total count of the CRs and collect the yaml for each of the backuprepository CR in your env.

I guess there should be quite lots of backuprepository CRs, because each volume to be backed up seems to belong to different namespace, something like `ora-0e07712`, so each volume matches to a different backuprepository since the namespace is different.

**Comment by Lyndon-Li** [comment]
@pdstefan Meanwhile, you can try the Kopia path to see if the problem can be solved. To do this, add `uploader-type=kopia` flag during installation or follow `[this guide](https://velero.io/docs/v1.10/file-system-backup/#:~:text=integration%20for%20details.-,Path%20selection,-Velero%E2%80%99s%20FSB%20supports)`

**Comment by pdstefan** [comment]
> @pdstefan Looks like the problem is related to your env, and we need to know what happened to the backup repo connection. Please do below steps and share more info to us:
> 
>     1. If you can reproduce the problem, please set Velero's log level to `debug` and recollect the log. The rep connection logs are all in debug level. You can follow [this guide](https://velero.io/docs/v1.10/troubleshooting/#:~:text=more%20usage%20details.-,Getting%20velero%20debug%20logs,-You%20can%20increase) to enable the debug log.
> 
>     2. Please share the info of backuprepository CRs in your env, specifically, collect the total count of the CRs and collect the yaml for each CR in your env.
> 
> 
> I guess there should be quite lots of backuprepository CRs, because each volume to be backed up seems to belong to different namespace, something like `ora-0e07712`, so each volume matches to a different backuprepository since the namespace is different.

1. When this happens again, I`ll start velero in debug mode. At this time the environment is stable and we are not able to reproduce the behavior. 
2.  Correct, we backup each namespace in a different repository. 
 ``` 
kubectl -n velero get backuprepositories.velero.io | wc -l 
1054
```

**Comment by pdstefan** [comment]
> @pdstefan Meanwhile, you can try the Kopia path to see if the problem can be solved. To do this, add `--uploader-type=kopia` flag during installation or follow [this guide](https://velero.io/docs/v1.10/file-system-backup/#:~:text=integration%20for%20details.-,Path%20selection,-Velero%E2%80%99s%20FSB%20supports).

I`ll try Kopia on another environment, unfortunately the env running restic is our production environment (not the best place to experiment). 

If you see any improvements that could be done in an env backing up 1000+ namespaces, please do share.

**Comment by Lyndon-Li** [comment]
Some more clue:
Some backup repo CR was handled by the controller long time after the requesting backup had failed:

```
time="2023-02-08T12:01:28+01:00" level=info msg="1 errors encountered backup up item" backup=velero/backup-ora-0d01676-20230207210048 logSource="pkg/backup/backup.go:421" name=ora-0d01676-65cc5f85cf-ppvz9
time="2023-02-08T12:01:28+01:00" level=error msg="Error backing up item" backup=velero/backup-ora-0d01676-20230207210048 error="failed to wait BackupRepository: timed out waiting for the condition" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/repository/ensurer.go:129" error.function="github.com/vmware-tanzu/velero/pkg/repository.(*RepositoryEnsurer).createBackupRepositoryAndWait" 
...
time="2023-02-08T12:01:59+01:00" level=info msg="Initializing backup repository" backupRepo=velero/ora-0d01676-default-restic-rf5gg logSource="pkg/controller/backup_repository_controller.go:113"
```

**Comment by Lyndon-Li** [comment]
By analyzing the logs, I could guess what happened in the env, but to confirm this, we still need to reproduce the problem in the env and check the debug log @pdstefan 

See below details:

- backupRepository CR is processed by backupRepository controller, for which the concurrency is 1. This means the controller processes the CRs sequentially
- Every 5 minutes, all the existing CRs are re-enqueued for the controller to do initialization, stale lock removal, maintenance, and re-connect. Therefore, for example, in the current env, there are 1054 CRs, so every 5 mins, these 1054 CRs will be processed by the controller one after one
- There is no prioritizing of the CR queue, that is, whenever a new CR is created, it is put in the tail of the queue. Therefore, worstly, the new CR needs to wait more than 1000 existing CRs
- When a backup creates a CR, it wait the CR to become ready until the timeout, 5 mins
- For each CR processing, the controller may do below things:

  1.      Maintenance: This is not executed every time, but only when the maintenance is due for a CR. The time to finish this is XX minutes~X hours
  2.     Remove stale repo lock: This is execute every time for a Ready CR. The time for this is X seconds
  3.     Reconnect repo: This is execute if the repo is not in Ready state. The time for this is X seconds

As a calculation, we can ignore point 3 as it is for abnormal repo CRs only. For 1 and 2:

        - Maintenance doesn't happen every time, but once it happens, it will take very long time and during this, all the CRs will be blocked
        - Remove stale repo lock happens every time for Ready repo, say we have 1000 repo CRs, and every operations takes 1s, in the worst case, it takes 1000 * 1s = 17min for a new CR in the tail to be processed. This is far more than the backup wait timeout 5 min

Conclusively, this problem happens only in the env that there are lots of backupRepository CRs

**Comment by Lyndon-Li** [comment]
We have two directions to make the fix/enhancement:
1. In short-term,

**Comment by Lyndon-Li** [comment]
@pdstefan To answer your below question:

> If you see any improvements that could be done in an env backing up 1000+ namespaces, please do share

According to the above analysis, there is indeed a improvement for the current problem if you switch to Kopia path:

- Maintenance: Kopia's maintenance is much faster than Restic's
- Remove stale repo lock: Kopia doesn't have a repo lock, so we never need to deal with any stale locks in the repo. This means, for this operation, Kopia path doesn't nothing

**Comment by Lyndon-Li** [comment]
@pdstefan Meanwhile, could you share more info about your env?
We see the workloads in different namespaces seem to be in the same category, specifically, then are all named as ora-xxx. So why do you need to creates so may homogeneous workloads in different namespaces?

We ask this because if the workloads are not in its own namespaces, there will not be so many backupRepository CRs.

**Comment by pdstefan** [comment]
Hi,  
Thank you for the in-depth analysis. We will consider trying out and maybe switching to Kopia in the near future. 


We have one application which we customize for every customer (ex: different resources, ingress urls, config maps, etc). Each namespace is a different customer and we have both scheduled and realtime backups for each namespace.  Does this explanation explain/justify the large number of backupRepository CRs?

**Comment by Lyndon-Li** [comment]
Looks like you are using namespaces to simulate a multiple tenant environment.  
Velero has not support multiple tenant yet, if Velero finally defines multiple tenant based on this multiple namespaces pattern, Velero will not define backuprepositories by source namespaces.

**Comment by pdstefan** [comment]
As it turns out, I managed to reproduce the initial situation. 
We are using Minio deployed in kube as S3 provider and during the night the PVC filled up. We added more disk in the morning, but multiple operations are "Stalled".  
- 100+ of the scheduled backups are in status "New"
``` 
velero backup get | grep "New" | wc -l
103
``` 
- backups with status "New" are only processed when the Velero pod starts up. We left the pod running for 20+ minutes and  none of the "New" backups are processed. 

At this point I will continue deleting the velero pod every  10 or so minutes to force the scheduled backups to complete. What I once again find weird is that if I want to take a manual backup now, it will be queued and only processed all backups in status "New" move to either Completed or Failed. 

Logs from: `velero debug`  [here](https://transfer.kronsoft.cloud/G5fucA/bundle-2023-03-21-12-24-30.tar.gz)
